src/main/java/org/opensearch/security/dlic/rest/api/AccountApiAction.java

```diff
@@ -61,10 +63,13 @@
  * Currently this action serves GET and PUT request for /_opendistro/_security/api/account endpoint
  */
 public class AccountApiAction extends AbstractApiAction {
+    static final String SAVED_TENANT = "saved_tenant";
```

Comment:
keep uniformity - `private` static final String

Reply:
Will be changed; thanks